### PR TITLE
Fix/duplicated getting started content

### PR DESCRIPTION
**Pull request by ahosgood**

### CODE REVIEW

File: src/get-started/index.md.njk [hunk 1]
```diff
@@ -8,3 +8,7 @@ show_subnav: true
 This section will enable you to get up and running as quickly as possible. Find easy access to toolkits, guides and helpful information on how to get setup, to start designing at HM Land Registry.
 
 To make your service consistent with GOV.UK, you should start by checking what exists in the [GOV.UK Design System](https://design-system.service.gov.uk).
+
```
Comment (avflay) [inline src/get-started/index.md.njk:11]: Can we add "Refer to the prototyping and developing setup guides." here please?

File: src/get-started/index.md.njk [hunk 2]
```diff
@@ -8,3 +8,7 @@ show_subnav: true
...
+## Feedback
+
+Is there something missing from this section that you would find useful? Contact HM Land Registry DesignOps. Your feedback will help us to improve it.
```
Comment (avflay) [inline src/get-started/index.md.njk:14]: Also, change Contact HM Land Registry DesignOps to "Contact HMLR-Design-System-Support@landregistry.gov.uk."

**Review by avflay — CHANGES_REQUESTED** [review]

File: src/get-started/index.md.njk (hunk 1 above)
Comment (ahosgood) [reply src/get-started/index.md.njk:11]: I know the "prototyping" in this refers to https://hmlr-design-fix-duplica-zhyepi.herokuapp.com/get-started/prototyping/ but what does the "developing setup guides" refer to?

File: src/get-started/index.md.njk (hunk 2 above)
Comment (ahosgood) [reply src/get-started/index.md.njk:14]: What about linking to the "get in touch" page? https://hmlr-design-fix-duplica-zhyepi.herokuapp.com/get-in-touch/

File: src/get-started/index.md.njk (hunk 1 above)
Comment (avflay) [reply src/get-started/index.md.njk:11]: Apologies, it should say "Refer to the prototyping and production setup guides."

File: src/get-started/index.md.njk (hunk 2 above)
Comment (avflay) [reply src/get-started/index.md.njk:14]: yep, that's perfect

File: src/get-started/index.md.njk (hunk 1 above)
Comment (ahosgood) [reply src/get-started/index.md.njk:11]: Changed

File: src/get-started/index.md.njk (hunk 2 above)
Comment (ahosgood) [reply src/get-started/index.md.njk]: Done

**Review by avflay — APPROVED** [review]

**Pull request merged**